build/nse/ucnf-kiknos/README.md

```diff
@@ -94,12 +94,12 @@ Makefile consists of the following rules:
 Makefile options:
 
 - `CLUSTER` - Set the cluster name - (Default: `kiknos-demo-1`)
-- `ORG` - Set the org of new built image - (Default: `tiswanso`)
+- `ORG` - Set the org of new built image - (Default: `cisco-app-networking`)
```

Comment:
I meant to bring up this discussion.  AFAIK, there's no docker.io/cisco-app-networking ORG.  I see you guys created `ciscolabs` with the kiknos work.  Should we start using that for our images created from `github.com/cisco-app-networking/*` as well?

Reply:
Yeah, good question. I guess the only disadvantage of `ciscolabs` is that it is not really an organisation, but an user :(
So everybody needs to share the same login credentials. Is that right, @brecode ?

Reply:
Correct, I can turn the account into an organization and we can use it for our docker images. @tiswanso, let me know if that's something we want to do

Reply:
I went ahead and created "ciscoappnetworking" org in docker.io.  Will add a follow-on change to push the images there